.github/workflows/test.yml

```diff
@@ -717,6 +717,20 @@ jobs:
           cache: true
           manifest-path: test/pyproject-manifest/pyproject.toml
 
+  environment-activation:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Move pixi.toml
+        run: mv test/default/* .
+      - uses: ./
+        with:
+          activate-environment: true
+      - run: which python | grep '.pixi/envs/default/bin/python'
```

Comment:
Could you make sure that `bash`, `cmd.exe` and `powershell` work on windows? They might behave differently depending on \\ or /